### PR TITLE
add parquet schemas to autopush

### PR DESCRIPTION
**Pull request by relud**
r? @bbangert 

I converted these schemas from https://github.com/mozilla-services/push_derived/tree/master/hindsight/hs_run/output

### CODE REVIEW

File: parquet_schemas/autopush.autoendpoint.docker.autoendpoint/fallback.cfg.disabled [hunk 1]
```diff
@@ -0,0 +1,30 @@
+-- vim:ft=lua
+message_matcher = message_matcher .. " && (Fields[Fields.router_key] != 'simplepush'"
+message_matcher = message_matcher .. " && Fields[Fields.router_key] != 'webpush'"
+message_matcher = message_matcher .. " || Fields[Fields.remote_ip] == NIL)"
```
Comment (jrconlin) [inline parquet_schemas/autopush.autoendpoint.docker.autoendpoint/fallback.cfg.disabled]: See above re: additional router_keys to match.

File: parquet_schemas/autopush.autoendpoint.docker.autoendpoint/endpoint.cfg [hunk 2]
```diff
@@ -0,0 +1,30 @@
+-- vim:ft=lua
+message_matcher = message_matcher .. " && (Fields[Fields.router_key] == 'webpush' || Fields[Fields.router_key] == 'simplepush') && Fields[remote_ip] != NIL"
```
Comment (jrconlin) [inline parquet_schemas/autopush.autoendpoint.docker.autoendpoint/endpoint.cfg]: Should we add router_keys for 'gcm', 'fcm' (for android) and 'apns' (for iOS)?

**Comment by bbangert** [comment]
This should be updated to result in a single parquet schema output for a single table.

File: parquet_schemas/autopush.autopush.docker.autopush/conn.cfg [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    optional binary fields_remote_ip (UTF8);
+    optional binary fields_uaid_hash (UTF8);
+    optional binary fields_message (UTF8);
+    optional boolean fields_error;
+    optional binary fields_user_agent (UTF8);
+    optional binary fields_channelid (UTF8);
+    optional binary fields_message_id (UTF8);
+    optional int64 fields_message_size;
+    optional binary fields_message_source (UTF8);
+    optional binary fields_browser_os (UTF8);
```
Comment (pjenvey) [inline parquet_schemas/autopush.autopush.docker.autopush/conn.cfg]: browser_os/version looks wrong, we log:

```
    "ua_browser_ver": "36.0",
    "ua_browser_family": "Firefox",
    "ua_os_family": "Windows 8.1",
    "ua_os_ver": "",
```

**Comment by pjenvey** [comment]
I think we'd also want `optional binary tls_auth (UTF8);` on endpoint (for the upcoming partner endpoint)

File: parquet_schemas/autopush.autopush.docker.autopush/conn.cfg (hunk 3 above)
Comment (relud) [inline parquet_schemas/autopush.autopush.docker.autopush/conn.cfg]: fields_browser_os and fields_browser_version are being created by line 11 as user agent transforms from `"user_agent"`

**Comment by bbangert** [comment]
Will be pulling this and using it as the basis for a new PR, issue #874.

**Pull request closed without merging**